src/Transports/MassTransit.ActiveMqTransport/ActiveMqTransport/Middleware/ActiveMqConsumer.cs

```diff
@@ -77,39 +68,17 @@ void HandleMessage(IMessage message)
             }, Stopping);
         }
 
-        Task HandleDeliveryComplete()
-        {
-            if (IsStopping)
-                _deliveryComplete.TrySetResult(true);
-
-            return Task.CompletedTask;
-        }
-
         protected override Task StopAgent(StopContext context)
```

Comment:
Might be worth having a virtual method called when the consumer needs to shut off transport message delivery, vs. overloading StopAgent and calling base.